### PR TITLE
Quick fix for new Bind poller in agent mode

### PR DESCRIPTION
**Pull request by slashdoom**
$name and $app_id were not defined when using new bind poller in agent mode leading to NaN/U RRD data.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/polling/applications/bind.inc.php [hunk 1]
```diff
@@ -2,13 +2,11 @@
 
 use LibreNMS\RRD\RrdDefinition;
 
-echo ' bind ';
-
+$name = 'bind';
```
Comment (murrant) [inline includes/polling/applications/bind.inc.php]: You have removed the echo.  I suggest this:
`echo " $name";`

Comment (slashdoom) [inline includes/polling/applications/bind.inc.php]: Thanks, I didn't realize that was preferred as it seems kind of inconsistent in these inc files.  But I've added back in as suggested.

File: includes/polling/applications/bind.inc.php [hunk 2]
```diff
@@ -2,20 +2,20 @@
...
-    $name = 'bind';
-    $app_id = $app['app_id'];
     $options = '-O qv';
     $mib          = 'NET-SNMP-EXTEND-MIB';
     $oid     = 'nsExtendOutputFull.4.98.105.110.100';
     $bind    = snmp_get($device, $oid, $options, $mib);
 }
 update_application($app, $bind);
 
+echo " $name";
```
Comment (laf) [inline includes/polling/applications/bind.inc.php]: Pop this back up the top, it needs to be echoed before we do anything for debugging.

Comment (slashdoom) [inline includes/polling/applications/bind.inc.php]: Ha, did I get it this time?  I used the apache, nginx and mysql files as examples so I'm a bit confused still.

**Pull request merged**